slack_bolt/authorization/authorize.py

```diff
@@ -110,27 +111,69 @@ def __call__(
         team_id: str,
         user_id: Optional[str],
     ) -> Optional[AuthorizeResult]:
-        bot: Optional[Bot] = self.installation_store.find_bot(
-            enterprise_id=enterprise_id, team_id=team_id,
-        )
-        if bot is None:
-            self.logger.debug(
-                f"No installation data found "
-                f"for enterprise_id: {enterprise_id} team_id: {team_id}"
+
+        # couldn't get this to work
+        # installation_or_bot: Optional[Bot] | Optional[Installation]
+
+
+        # TODO
```

Comment:
I haven't been able to test this yet since I can't get my local OAuth flow to work (i think my setup is messed up). Even without these changes it is hanging. I will try to sort that out but wanted to post this here so you can build upon it @seratch